test/fixtures/google-gax-packaging-test-app/test/gapic-v1beta1.js

```diff
@@ -177,6 +177,45 @@ describe('EchoClient', () => {
       });
     });
 
+    it('invokes pagedExpand using async iterator', async () => {
```

Comment:
Just to make sure, this file was originally auto-generated. We will need to have similar auto-generated code in gapic-generator-typescript test.

Reply:
added generated test code in pr: https://github.com/googleapis/gapic-generator-typescript/pull/207